pkg/olm/catalogsource.go

```diff
@@ -0,0 +1,134 @@
+package olm
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	oplmV1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CatalogSourceBuilder provides a struct for catalogsource object
+// from the cluster and a catalogsource definition.
+type CatalogSourceBuilder struct {
+	// CatalogSourceBuilder definition. Used to create
```

Comment:
Please update it to CatalogSource definition. Under Definition we keep CatalogSource only.